client/blocks/plan-thank-you-card/style.scss

```diff
@@ -53,6 +53,15 @@
 	border-width: 1px 1px 2px;
 }
 
+.plan-thank-you-card__button.is-placeholder,
+.plan-thank-you-card__button.is-placeholder:visited {
+	@include placeholder();
+	pointer-events: none;
```

Comment:
Why do we need to turn off pointer events? Did we want something to be clickable underneath?